java/ingest/ingest-status-store/src/main/java/sleeper/ingest/status/store/job/DynamoDBIngestJobStatusStore.java

```diff
@@ -124,6 +126,15 @@ public void jobFinished(IngestJobFinishedEvent event) {
         }
     }
 
+    @Override
+    public void jobFailed(IngestJobFailedEvent event) {
+        try {
+            save(createJobFailedUpdate(event, jobUpdateBuilder(event.getTableId(), event.getJobId())));
+        } catch (RuntimeException e) {
+            throw new IngestStatusStoreException("Failed jobFailed for job " + event.getJobId(), e);
```

Comment:
Maybe this exception message should be something like `Failed saving jobFailed event for job {}`? Same could be done for the other exception messages in this class

Reply:
Done, thanks